src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -148,6 +147,8 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
   override def adjustInputPaths(callKey: CallKey, inputs: CallInputs, workflowDescriptor: WorkflowDescriptor): CallInputs = inputs mapValues gcsPathToLocal
   override def adjustOutputPaths(call: Call, outputs: CallOutputs): CallOutputs = outputs mapValues gcsPathToLocal
 
+  def fileHasher: FileHasher = { wdlFile: WdlFile => getCrc32c(GoogleCloudStoragePath(wdlFile.value)) }
```

Comment:
Correct but it's an implementation of the function defined in `Backend` which will be needed whenever you need a hash for a WdlValue.
Basically the idea behind this is that computing the hash for a WdlValue is almost self-contained, except for `WdlFile`s where it's backend-specific. This is why there is this implicit parameter in the `getHash` function, and also why every `Backend` has to implement its own way to get a has for a `WdlFile`.